frontend/src/metabase/lib/expressions/parser.js

```diff
@@ -358,7 +380,7 @@ export class ExpressionParser extends CstParser {
 
     $.RULE("parenthesisExpression", returnType => {
       $.CONSUME(LParen);
-      $.SUBRULE($.any, { LABEL: "expression", ARGS: [returnType] });
+      $.SUBRULE($.expression, { LABEL: "expression", ARGS: [returnType] });
```

Comment:
Unrelated to the change, but just for my own edification, what's an example of something in `any` but  outside `expression`?

Reply:
The previous grammar uses any to distinguish various top-level grammar rules and to restrict the types. This slowly goes away and at some point hopefully `any` will disappear completely.